internal/pkg/cli/svc_deploy.go

```diff
@@ -336,6 +340,16 @@ func workloadManifest(in *workloadManifestInput) (interface{}, error) {
 	if err != nil {
 		return nil, fmt.Errorf("apply environment %s override: %s", in.envName, err)
 	}
+	serviceInRegion, err := mft.IsServiceAvailableInRegion(in.targetEnv.Region)
```

Comment:
I feel like `IsServiceAvailableInRegion` this should be a method of our [deployer](https://github.com/aws/copilot-cli/blob/183c7302ca142997d618e91bec9e7947b03aacd2/internal/pkg/cli/svc_deploy.go#L115) rather than the manifest struct 🤔

Reply:
+1 to this!